experimental/gpu_monitoring/gpu_data_collector.py

```diff
@@ -140,6 +153,55 @@ def get_physicalhostname():
            return str(value[0])[2:][:-1]
 
 
+def get_counter_value(file_path):
+    file = open(file_path, "r")
+    value = file.read()
+    file.close()
+    return int(value)
+
+
+def get_counter_value(file_path):
```

Comment:
It looks like this function has been declared twice

Reply:
I removed the duplicate "get_counter_value" function.